test/validation/api/packet/packet.c

```diff
@@ -907,14 +907,23 @@ static void packet_test_length(void)
 
 static void packet_test_reset(void)
 {
-	uint32_t len, max_len, headroom;
+	uint32_t len, max_len, headroom = 128, tailroom;
 	uint32_t uarea_size = default_param.pkt.uarea_size;
 	uintptr_t ptr_len;
-	void *data, *new_data, *tail, *new_tail;
+	void *data, *new_data, *head, *tail, *new_tail;
 	struct udata_struct *udat;
 	odp_packet_t pkt;
+	odp_pool_t pool;
+	odp_pool_param_t pool_param;
 
-	pkt = odp_packet_alloc(default_pool, packet_len);
+	memcpy(&pool_param, &default_param, sizeof(odp_pool_param_t));
```

Comment:
Could just assign, `odp_pool_param_t pool_param = default_param`.

Reply:
Changed in V2.